### PR TITLE
Remove redundant code

### PR DESCRIPTION
**Pull request by RipulHandoo**
## Proposed changes

This pull request addresses the issue [ https://github.com/litmuschaos/litmus/issues/4206 ], which was about removing inefficient code that was causing redundant conditions in the codebase.

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
N/A

## Special notes for your reviewer:

N/A

### CODE REVIEW

File: chaoscenter/web/src/views/ExperimentDashboardV2/ExperimentDashboardV2Table.tsx [hunk 1]
```diff
@@ -208,15 +208,15 @@ const ExperimentDashboardV2Table = ({
               case ExperimentRunStatus.RUNNING:
                 return (
                   <RunExperimentButton
-                    buttonProps={{ disabled: lastExperimentRunStatus === ExperimentRunStatus.RUNNING }}
+                    buttonProps={{ disabled: true }}
```
Comment (gdsoumya) [inline chaoscenter/web/src/views/ExperimentDashboardV2/ExperimentDashboardV2Table.tsx:211]: I think we can completely remove this prop in the first place as I don't see it being used. Tagging @amityt / @Saranya-jena to confirm

**Review by gdsoumya — CHANGES_REQUESTED** [review]

File: chaoscenter/web/src/views/ExperimentDashboardV2/ExperimentDashboardV2Table.tsx (hunk 1 above)
Comment (RipulHandoo) [reply chaoscenter/web/src/views/ExperimentDashboardV2/ExperimentDashboardV2Table.tsx:211]: Is this alright @gdsoumya 
```
return (
                  <RunExperimentButton
                    experimentID={data.experimentID}
                    refetchExperiments={refetchExperiments}
                  />
                );
              case ExperimentRunStatus.QUEUED:
                return (
                  <RunExperimentButton
                    experimentID={data.experimentID}
                    refetchExperiments={refetchExperiments}
                  />
                );
```

Comment (Saranya-jena) [reply chaoscenter/web/src/views/ExperimentDashboardV2/ExperimentDashboardV2Table.tsx:211]: cc: @SahilKr24

**Comment by RipulHandoo** [comment]
@Saranya-jena I raised a new PR. This time all the checks have passed.

**Comment by Saranya-jena** [comment]
You have updated this branch right? @RipulHandoo

**Comment by RipulHandoo** [comment]
@Saranya-jena should I raise PR again.
    Yes

**Review by SarthakJain26 — APPROVED** [review]

File: chaoscenter/web/src/views/ExperimentDashboardV2/ExperimentDashboardV2Table.tsx (hunk 1 above)
Comment (RipulHandoo) [reply chaoscenter/web/src/views/ExperimentDashboardV2/ExperimentDashboardV2Table.tsx]: @arkajyotiMukherjee @gdsoumya @Saranya-jena 
I have shorten it to an if statement. 
PTAL!
```
if (lastExperimentRunStatus === ExperimentRunStatus.RUNNING || lastExperimentRunStatus === ExperimentRunStatus.QUEUED) {
  return (
    <RunExperimentButton
      buttonProps={{ disabled: true }}
      experimentID={data.experimentID}
      refetchExperiments={refetchExperiments}
    />
  );
} else {
  return (
    <RunExperimentButton 
      experimentID={data.experimentID} 
      refetchExperiments={refetchExperiments} 
    />
  );
}
```

**Review by gdsoumya — APPROVED** [review]

**Review by amityt — APPROVED** [review]

**Comment by amityt** [comment]
Hi @RipulHandoo , thanks for the contribution! There are some merge conflicts, can you please update your PR!

**Comment by RipulHandoo** [comment]
> Hi @RipulHandoo , thanks for the contribution! There are some merge conflicts, can you please update your PR!

@amityt I've solved the conflicts. 
PTAL!

**Review by hrishavjha — APPROVED** [review]
Changes looks good to me 🚀

**Review by arkajyotiMukherjee — APPROVED** [review]
Thanks for your contribution to the litmus repo @RipulHandoo 🚀

**Pull request merged**